### PR TITLE
Update

### PR DESCRIPTION
**Pull request by katcipis**

### CODE REVIEW

File: cv.tex [hunk 1]
```diff
@@ -462,15 +497,93 @@ \section{open source projects}
...
+
+%------------------------------------------------
+\begin{entrylist}
+\entry
+{2018}
+{Object Orientation in Go}
+{\href{http://www.thedevelopersconference.com.br/tdc/2018/florianopolis/trilha-golang}{The Developers Conference}}
+{
+
+For people that comes from a background on Java or other classic object oriented languages
```
Comment (vitorarins) [inline cv.tex]: ... people that *come from...

File: cv.tex [hunk 2]
```diff
@@ -462,15 +497,93 @@ \section{open source projects}
...
+%------------------------------------------------
+\begin{entrylist}
+\entry
+{2016}
+{Building Resilient Services in Go}
+{\href{https://2016.gopherconbr.org/en/}{GopherCon Brazil}}
+{
+
+Resilience is not about never failing, but how do you recover from it.
+How you can prevent your services from locking down or exhausting all its resources ?
```
Comment (vitorarins) [inline cv.tex]: *How can you ...

File: cv.tex [hunk 3]
```diff
@@ -462,15 +497,93 @@ \section{open source projects}
...
+How to perform graceful service degradation ? Can this kind of behaviour be tested properly ?\\
+
+On Go we have some new features, like Contexts, that helps us to model timeouts and cancellation properly.\\
+
+They can be combined with other useful features as select and channels to model timeouts and resource pools,
+which can be essential to provide proper service degradation instead of total failure of the system.\\
+
+On this talk I try to answer this questions using new features available on Go 1.7, direct from production ready software.\\
+
+Presentation source be found \href{https://github.com/katcipis/my.presentations/tree/master/resilient-services-in-go}{here}.
```
Comment (vitorarins) [inline cv.tex]: ... source *can be found ...

File: cv.tex [hunk 4]
```diff
@@ -462,15 +497,93 @@ \section{open source projects}
...
+\entry
+{2016}
+{Real Life Kubernetes}
+{\href{http://www.thedevelopersconference.com.br/tdc/2016/florianopolis/trilha-devops}{The Developers Conference}}
+{
+
+On this presentation we will give a short introduction on Kubernetes and show the experience of learning
+and using Kubernetes on production for two very distinct systems.\\
+
+The first one is a data acquisition system, involving running multiple instance of crawlers,
```
Comment (vitorarins) [inline cv.tex]: ... multiple *instances of a crawler,...

File: cv.tex [hunk 5]
```diff
@@ -462,15 +497,93 @@ \section{open source projects}
...
+
+The second one is a web application, involving network analysis using graphs with the ultimate goal of fraud prevention.
+The application is strongly bounded with the microservices architecture and the twelve factor app.\\
+
+Graph and document databases, cache layers, a message broker and a distributed filesystem are some of
+the technologies surrounding the application ecosystem.\\
+
+Also making a great case of integration between AWS and k8s using tools such as autoscaling.\\
+
+Presentation source be found \href{https://github.com/katcipis/my.presentations/tree/master/real-life-kubernetes}{here}.
```
Comment (vitorarins) [inline cv.tex]: .. *can be found...

**Review by vitorarins — APPROVED** [review]
I would definitely hire you :smiley: 

The comments are just the way I would right, not necessarily the correct way.
Also I would say your English is more than advanced, probably you can say is fluent without distinguishing reading, writing and speaking. But that is just my opinion.

File: cv.tex (hunk 5 above)
Comment (katcipis) [reply cv.tex]: Thanks for the help =)

**Pull request merged**